docker-compose.yml

```diff
@@ -11,6 +11,30 @@ services:
     volumes:
       - "./docker/sirius/openapi.yml:/tmp/openapi.yml"
 
+  experian-crosscore-mock:
+    image: stoplight/prism:5.11.1
+    command: mock -h 0.0.0.0 -p 4011 -d /tmp/openapi.yml
```

Comment:
FWIW this doesn't need to be on a different port to sirius-mock: they can both be on 4011 because it's not forwarded to the host machine